src/context/like-context.js

```diff
@@ -0,0 +1,63 @@
+import { createContext, useContext, useEffect } from "react";
+import { useAxios } from "../utils/useAxios";
+
+const LikeContext = createContext();
+
+const LikeProvider = ({ children }) => {
+   const { response: likedVideos, apiCall: getLikedVideos } = useAxios();
```

Comment:
You can change the name of the API call as you are posting, deleting, and not just getting.

Reply:
Done.